src/anomalib/models/cflow/anomaly_map.py

```diff
@@ -18,12 +19,12 @@ class AnomalyMapGenerator(nn.Module):
     def __init__(
         self,
         image_size: ListConfig | tuple,
-        pool_layers: list[str],
+        pool_layers: Sequence[str],
     ) -> None:
         super().__init__()
         self.distance = torch.nn.PairwiseDistance(p=2, keepdim=True)
         self.image_size = image_size if isinstance(image_size, tuple) else tuple(image_size)
-        self.pool_layers: list[str] = pool_layers
+        self.pool_layers: Sequence[str] = pool_layers
```

Comment:
Just curiosity, why is this changed to `Sequence`? CLI-related?

Reply:
I cannot assign default parameters if I keep the type as list. With Sequence, I can use both tuple and list